src/components/TaskList/TaskList.jsx

```diff
@@ -9,68 +9,103 @@ const TaskVisibility = require('../../models/TaskVisibility')
 class TaskList extends React.Component {
   constructor(props) {
     super(props)
-    this.state = { selectedIndex: null }
-    this.onKeyUp = this.onKeyUp.bind(this)
-    this.onKeyDown = this.onKeyDown.bind(this)
+    this.state = { focusedIndex: 0, showHidden: false }
   }
 
   componentDidMount() {
-    document.addEventListener('keyup', this.onKeyUp)
-    document.addEventListener('keydown', this.onKeyDown)
+    this.keyDownEventListener = event => this.onKeyDown(event)
+    document.addEventListener('keydown', this.keyDownEventListener)
   }
 
   componentWillUnmount() {
-    document.removeEventListener('keyup', this.onKeyUp)
-    document.removeEventListener('keydown', this.onKeyDown)
+    document.removeEventListener('keydown', this.keyDownEventListener)
   }
 
-  onSnoozeClick(event) {
-    event.currentTarget.blur() // defocus button
-    this.props.dispatch({ type: 'TASKS_SNOOZE' })
+  onKeyDown(event) {
+    const up = ['ArrowUp', 'k']
+    const down = ['ArrowDown', 'j']
+    const open = ['Enter', 'o']
+    const select = [' ', 'x']
```

Comment:
Yeah. It probably makes sense to add this to the config file, but for now I'm just hard coding them.